static/app/components/nav/primary.tsx

```diff
@@ -0,0 +1,411 @@
+
+interface SidebarItemProps {
+  item: NavSidebarItem;
+  children?: React.ReactNode;
+  onClick?: MouseEventHandler<HTMLElement>;
+}
+
+function SidebarBody({children}: {children: React.ReactNode}) {
+  return <SidebarItemList>{children}</SidebarItemList>;
+}
+
+function SidebarFooter({children}: {children: React.ReactNode}) {
+  return (
+    <SidebarFooterWrapper>
+      <SidebarItemList>{children}</SidebarItemList>
+    </SidebarFooterWrapper>
+  );
+}
+
+function SidebarItem({item}: SidebarItemProps) {
+  const to = resolveNavItemTo(item);
+  const SidebarChild = to ? SidebarLink : SidebarMenu;
+  const organization = useOrganization();
+
+  const FeatureGuard = item.feature ? Feature : Fragment;
+  const featureGuardProps: any = item.feature ?? {};
+
+  const recordAnalytics = useCallback(
+    () =>
+      trackAnalytics('growth.clicked_sidebar', {item: item.analyticsKey, organization}),
+    [organization, item.analyticsKey]
+  );
+
+  return (
+    <FeatureGuard {...featureGuardProps}>
+      <SidebarItemWrapper>
+        <SidebarChild item={item} key={item.label} onClick={recordAnalytics}>
+          {item.icon}
+          <span>{item.label}</span>
+        </SidebarChild>
+      </SidebarItemWrapper>
+    </FeatureGuard>
+  );
+}
+
+function SidebarMenu({item, children, onClick}: SidebarItemProps) {
+  if (!item.dropdown) {
+    throw new Error(
+      `Nav item "${item.label}" must have either a \`dropdown\` or \`to\` value!`
+    );
+  }
+  return (
+    <DropdownMenu
+      position="right-end"
+      trigger={(props, isOpen) => {
+        return (
+          <NavButton
+            {...props}
+            onClick={event => {
+              onClick?.(event);
+              props.onClick?.(event);
+            }}
+          >
+            <InteractionStateLayer hasSelectedBackground={isOpen} />
+            {children}
+          </NavButton>
+        );
+      }}
+      items={item.dropdown}
+    />
+  );
+}
+
+function SidebarLink({children, item, onClick}: SidebarItemProps) {
+  const location = useLocation();
+  const isActive = isNavItemActive(item, location);
+  const isSubmenuActive = isSubmenuItemActive(item, location);
+  const to = resolveNavItemTo(item);
+  if (!to) {
+    throw new Error(
+      `Nav item "${item.label}" must have either a \`dropdown\` or \`to\` value!`
+    );
+  }
+  const linkProps = makeLinkPropsFromTo(to);
+
+  return (
+    <NavLink
+      {...linkProps}
+      onClick={onClick}
+      className={isActive || isSubmenuActive ? 'active' : undefined}
+      aria-current={isActive ? 'page' : undefined}
+    >
+      <InteractionStateLayer hasSelectedBackground={isActive || isSubmenuActive} />
+      {children}
+    </NavLink>
+  );
+}
+
+export function PrimaryNavigationItems() {
+  const organization = useOrganization();
+  const prefix = `organizations/${organization.slug}`;
+
+  return (
+    <Fragment>
+      <SidebarBody>
+        <SidebarItem
+          item={{
```

Comment:
Everything in this file was taken from `sidebar.tsx` except for the below items which used to be defined in a config file